lib/main.dart

```diff
@@ -17,9 +17,11 @@ Future<void> start() async {
     case '': // default to testchain
     case 'testchain':
       chain = TestSidechain();
+      break;
 
     case 'ethereum':
       chain = EthereumSidechain();
+      break;
```

Comment:
not that it matters, but this is not needed I'm pretty sure?

https://dart.dev/language/branches#switch-statements